tests/test_validators.py

```diff
@@ -0,0 +1,79 @@
+"""This is an example of json schema for the GDC using schemas defined
+in local yaml files (see gdcdictionary/examples).
+
+This test class is making sure the example jsons comply with the schema.
+
+Note this is NOT testing that the schema is sane. Just that we adhere
+to it
+
+"""
+try:
+    from importlib.resources import files
+except ImportError:
+    from importlib_resources import files
+
+import json
+import pytest
+
+import gdcdictionary
+
+
+def get_all_paths(subdir: str) -> str:
+    with files("tests") as path:
+        examples_path = path.parent / f"examples/{subdir}"
+        yield from sorted(examples_path.glob("*.json"))
+
+
+@pytest.mark.parametrize("path", get_all_paths("valid"))
+def test_valid_examples(path, schema):
+    with open(path) as f:
+        doc = json.load(f)
+        assert len(gdcdictionary.validate(doc)) == 0
+
+
+@pytest.mark.parametrize("path", get_all_paths("invalid"))
+def test_invalid_examples(path, schema):
+    with open(path) as f:
+        doc = json.load(f)
+        violations = gdcdictionary.validate(doc)
+        assert len(violations) > 0
+        print(violations)
```

Comment:
Is the print still needed?

Reply:
Its ok to leave it for debug purposes

Reply:
Agreed, it shouldn't cause issues leaving it in there. But in terms of our automated testing guidelines, would this be considered a "relevant detail"? This is an optional suggestion of course so feel free to mark the conversation as resolved as I've already approved.